MapboxNavigation/RouteMapViewController.swift

```diff
@@ -315,6 +316,10 @@ class RouteMapViewController: UIViewController {
         statusView.show(title, showSpinner: true)
     }
     
+    @objc func rerouteDidFail(notification: NSNotification) {
+        statusView.hide()
```

Comment:
Ideally, we'd only hide a status view of type reroute. Currently, any and all status updates will be hidden.

Reply:
Still planning to address this detail in this PR? Do we have a way of asking the status view what kind of message it’s displaying?

Reply:
I think this would require a larger refactor.

Reply:
Let’s either take care of that refactor here or ticket it out. This need has come up multiple times in the past. I think it would be quite unexpected that a request failure would blow away a “Simulating Navigation” banner, for instance.

Reply:
Let’s either take care of that refactor here or ticket it out. This need has come up multiple times in the past. I think it would be quite unexpected that a request failure would blow away a “Simulating Navigation” banner, for instance.